core/metrics-model/src/main/java/org/hawkular/metrics/model/NumericBucketPoint.java

```diff
@@ -29,57 +28,60 @@
  * @author Thomas Segismont
  */
 public class NumericBucketPoint extends BucketPoint {
-    private final double min;
-    private final double avg;
-    private final double median;
-    private final double max;
-    private final double sum;
-    private int samples;
+    private final Double min;
+    private final Double avg;
+    private final Double median;
+    private final Double max;
+    private final Double sum;
+    private Integer samples;
```

Comment:
If we keep it as an int, then it will be included in the JSON response. I changed it to Integer so that it is excluded. I don't see the need to include it when the bucket is empty.